src/fsharp/service/SemanticClassification.fsi

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.SourceCodeServices
+
+[<RequireQualifiedAccess>]
+type SemanticClassificationType =
```

Comment:
Could you add `///` comments to all of these public things please?

Reply:
Sure